taskmanager/src/Domain/Process/TaskService.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Domain\Process;
+
+use App\Domain\Ports\DTO\NewTask;
+use App\Domain\Ports\Incoming\AddTask;
+use App\Domain\Ports\Incoming\AssignTask;
+use App\Domain\Ports\Incoming\ChangeStatus;
+use App\Domain\Ports\Outgoing\TaskRepository;
+use App\Domain\Ports\Outgoing\UserRepository;
+use App\Domain\Process\Task\Status;
+use App\Domain\Process\Task\Task;
+use App\Domain\Process\User\User;
+
+class TaskService implements AddTask, AssignTask, ChangeStatus
```

Comment:
I have some doubts about one service that do everything related to task. What about SRP, is it ok?

Reply:
Yes I agree with you. I don't want do this more complicate so temporary I move this to one class.